strings/characters_unique.py

```diff
@@ -0,0 +1,53 @@
+#!python3
+"""
+The problem is to find a given string has unique characters.
+>>> isUniqueSet("Brick")
+True
+>>> isUniqueHash("Brick")
+True
+
+"""
+
+
+def isUniqueSet(string: str) -> bool:
+    """
+    This function uses a set to determine the string contains unique characters
+    >>> isUniqueSet("train")
+    True
+    >>> isUniqueSet("test")
+    False
+    """
+
+    hashSet = set()
```

Comment:
We can use `set` in different way. We can just `return len(list) == len(set) <- pseudocode' to determine unique characters.